cmake/process_options.cmake

```diff
@@ -117,3 +124,38 @@ if(ENABLE_BLAS)
 else()
     message(STATUS "ENABLE_BLAS is OFF.")
 endif()
+
+if(ENABLE_KOKKOS)
+    # Setting the Serial device for all cases.
+    option(Kokkos_ENABLE_SERIAL  "Enable Kokkos SERIAL device" ON)
+    message(STATUS "KOKKOS SERIAL DEVICE ENABLED.")
+
+    option(Kokkos_ENABLE_COMPLEX_ALIGN "Enable complex alignment in memory" OFF)
+
+    set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+    include(FetchContent)
+
+    FetchContent_Declare(kokkos
+                         GIT_REPOSITORY https://github.com/kokkos/kokkos.git
+                         GIT_TAG        3.6.00
+    )
+  
+    FetchContent_MakeAvailable(kokkos)
+
+    get_target_property(_kokkos_INC_DIR kokkos INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
Why use the underscore here?

Reply:
It is not necessary. I will remove it.

Reply:
It came back somehow. I will remove it, again.